src/history/hs.c

```diff
@@ -765,7 +765,6 @@ __wt_find_hs_upd(
     const uint8_t *recnop;
     int cmp;
     bool modify;
-
```

Comment:
I think the style guide is still to keep a blank line between declarations and initializations.